# Rosalind.info Problems and Solutions

This is a summary of my solutions to the problems on [rosalind.info](https://www.rosalind.info). The solution approaches of each problem are usually at the beginning of the corresponding source file. I will keep updating the list. Please feel free to share any of your thoughts!



|  #  | Problems and Solutions 
|------------- | ------------- 
| DNA  | [Counting DNA Nucleotides][dna]                                        \[ [solution][dna_py] \]
| RNA  | [Transcribing DNA into RNA][rna]                                       \[ [solution][rna_py] \]
| REVC | [Complementing a Strand of DNA][revc]                                  \[ [solution][revc_py] \]
| HAMM | [Counting Point Mutations][hamm]                                       \[ [solution][hamm_py] \]
| PROT | [Translating RNA into Protein][prot]                                   \[ [solution][prot_py] \]
| SUBS | [Finding a Motif in DNA][subs]                                         \[ [solution][subs_py] \]
| PERM | [Enumerating Gene Orders][perm]                                        \[ [solution][perm_py] \]
| PRTM | [Calculating Protein Mass][prtm]                                       \[ [solution][prtm_py] \]
| FIB  | [Rabbits and Recurrence Relations][fib]                                \[ [solution][fib_py] \]
| GC   | [Computing GC Content][gc]                                             \[ [solution][gc_py] \]
| IPRB | [Mendel's First Law][iprb]                                             \[ [solution][iprb_py] \]
| LCSM | [Finding a Shared Motif][lcsm]                                         \[ [solution][lcsm_py] \]
| MPRT | [Finding a Protein Motif][mprt]                                        \[ [solution][mprt_py] \]
| MRNA | [Inferring mRNA from Protein][mrna]                                    \[ [solution][mrna_py] \]
| ORF  | [Open Reading Frames][orf]                                             \[ [solution][orf_py] \]
| REVP | [Locating Restriction Sites][revp]                                     \[ [solution][revp_py] \]
| SPLC | [RNA Splicing][splc]                                                   \[ [solution][splc_py] \]
| LEXF | [Enumerating k-mers Lexicographically][lexf]                           \[ [solution][lexf_py] \]
| SIGN | [Enumerating Oriented Gene Orderings][sign]                            \[ [solution][sign_py] \]
| CONS | [Consensus and Profile][cons]                                          \[ [solution][cons_py] \]
| FIBD | [Mortal Fibonacci Rabbits][fibd]                                       \[ [solution][fibd_py] \]
| GRPH | [Overlap Graphs][grph]                                                 \[ [solution][grph_py] \]
| IEV  | [Calculating Expected Offspring][iev]                                  \[ [solution][iev_py] \]
| LIA  | [Independent Alleles][lia]                                             \[ [solution][lia_py] \]
| LGIS | [Longest Increasing Subsequence][lgis]                                 \[ [solution][lgis_py] \]
| LONG | [Genome Assembly as Shortest Superstring][long]                        \[ [solution][long_py] \]
| PMCH | [Perfect Matchings and RNA Secondary Structures][pmch]                 \[ [solution][pmch_py] \]
| PPER | [Partial Permutations][pper]                                           \[ [solution][pper_py] \]
| PROB | [Introduction to Random Strings][prob]                                 \[ [solution][prob_py] \]
| SSEQ | [Finding a Spliced Motif][sseq]                                        \[ [solution][sseq_py] \]
| TRAN | [Transitions and Transversions][tran]                                  \[ [solution][tran_py] \]
| TREE | [Completing a Tree][tree]                                              \[ [solution][tree_py] \]
| CAT  | [Catalan Numbers and RNA Secondary Structures][cat]                    \[ [solution][cat_py] \]
| CORR | [Error Correction in Reads][corr]                                      \[ [solution][corr_py] \]
| INOD | [Counting Phylogenetic Ancestors][inod]                                \[ [solution][inod_py] \]
| KMER | [k-Mer Composition][kmer]                                              \[ [solution][kmer_py] \]
| KMP  | [Speeding Up Motif Finding][kmp]                                       \[ [solution][kmp_py] \]
| LCSQ | [Finding a Shared Spliced Motif][lcsq]                                 \[ [solution][lcsq_py] \]
| LEXV | [Ordering Strings of Varying Length Lexicographically][lexv]           \[ [solution][lexv_py] \]
| MMCH | [Maximum Matchings and RNA Secondary Structures][mmch]                 \[ [solution][mmch_py] \]
| PDST | [Creating a Distance Matrix][pdst]                                     \[ [solution][pdst_py] \]
| REAR | [Reversal Distance][rear]                                              \[ [solution][rear_py] \]
| RSTR | [Matching Random Motifs][rstr]                                         \[ [solution][rstr_py] \]
| SSET | [Counting Subsets][sset]                                               \[ [solution][sset_py] \]
| ASPC | [Introduction to Alternative Splicing][aspc]                           \[ [solution][aspc_py] \]
| EDIT | [Edit Distance][edit]                                                  \[ [solution][edit_py] \]
| EVAL | [Expected Number of Restriction Sites][eval]                           \[ [solution][eval_py] \]
| MOTZ | [Motzkin Numbers and RNA Secondary Structures][motz]                   \[ [solution][motz_py] \]
| NWCK | [Distances in Trees][nwck]                                             \[ [solution][nwck_py] \]
| SCSP | [Interleaving Two Motifs][scsp]                                        \[ [solution][scsp_py] \]
| SETO | [Introduction to Set Operations][seto]                                 \[ [solution][seto_py] \]
| SORT | [Sorting by Reversals][sort]                                           \[ [solution][sort_py] \]
| SPEC | [Inferring Protein from Spectrum][spec]                                \[ [solution][spec_py] \]
| TRIE | [Introduction to Pattern Matching][trie]                               \[ [solution][trie_py] \]
| CONV | [Comparing Spectra with the Spectral Convolution][conv]                \[ [solution][conv_py] \]
| CTBL | [Creating a Character Table][ctbl]                                     \[ [solution][ctbl_py] \]
| DBRU | [Constructing a De Bruijn Graph][dbru]                                 \[ [solution][dbru_py] \]
| EDTA | [Edit Distance Alignment][edta]                                        \[ [solution][edta_py] \]
| FULL | [Inferring Peptide from Full Spectrum][full]                           \[ [solution][full_py] \]
| INDC | [Independent Segregation of Chromosomes][indc]                         \[ [solution][indc_py] \]
| ITWV | [Finding Disjoint Motifs in a Gene][itwv]                              \[ [solution][itwv_py] \]
| LREP | [Finding the Longest Multiple Repeat][lrep]                            \[ [solution][lrep_py] \]
| NKEW | [Newick Format with Edge Weights][nkew]                                \[ [solution][nkew_py] \]
| RNAS | [Wobble Bonding and RNA Secondary Structures][rnas]                    \[ [solution][rnas_py] \]
| AFRQ | [Counting Disease Carriers][afrq]                                      \[ [solution][afrq_py] \]
| CSTR | [Creating a Character Table from Genetic Strings][cstr]                \[ [solution][cstr_py] \]
| CTEA | [Counting Optimal Alignments][ctea]                                    \[ [solution][ctea_py] \]
| CUNR | [Counting Unrooted Binary Trees][cunr]                                 \[ [solution][cunr_py] \]
| GLOB | [Global Alignment with Scoring Matrix][glob]                           \[ [solution][glob_py] \]
| PCOV | [Genome Assembly with Perfect Coverage][pcov]                          \[ [solution][pcov_py] \]
| PRSM | [Matching a Spectrum to a Protein][prsm]                               \[ [solution][prsm_py] \]
| QRT  | [Quartets][qrt]                                                        \[ [solution][qrt_py] \]
| SGRA | [Using the Spectrum Graph to Infer Peptides][sgra]                     \[ [solution][sgra_py] \]
| SUFF | [Encoding Suffix Trees][suff]                                          \[ [solution][suff_py] \]
| CHBP | [Character-Based Phylogeny][chbp]                                      \[ [solution][chbp_py] \]
| CNTQ | [Counting Quartets][cntq]                                              \[ [solution][cntq_py] \]
| EUBT | [Enumerating Unrooted Binary Trees][eubt]                              \[ [solution][eubt_py] \]
| GASM | [Genome Assembly Using Reads][gasm]                                    \[ [solution][gasm_py] \]
| GCON | [Global Alignment with Constant Gap Penalty][gcon]                     \[ [solution][gcon_py] \]
| LING | [Linguistic Complexity of a Genome][ling]                              \[ [solution][ling_py] \]
| LOCA | [Local Alignment with Scoring Matrix][loca]                            \[ [solution][loca_py] \]
| MEND | [Inferring Genotype from a Pedigree][mend]                             \[ [solution][mend_py] \]
| MGAP | [Maximizing the Gap Symbols of an Optimal Alignment][mgap]             \[ [solution][mgap_py] \]
| MREP | [Identifying Maximal Repeats][mrep]                                    \[ [solution][mrep_py] \]
| MULT | [Multiple Alignment][mult]                                             \[ [solution][mult_py] \]
| PDPL | [Creating a Restriction Map][pdpl]                                     \[ [solution][pdpl_py] \]
| ROOT | [Counting Rooted Binary Trees][root]                                   \[ [solution][root_py] \]
| SEXL | [Sex-Linked Inheritance][sexl]                                         \[ [solution][sexl_py] \]
| SPTD | [Phylogeny Comparison with Split Distance][sptd]                       \[ [solution][sptd_py] \]
| WFMD | [The Wright-Fisher Model of Genetic Drift][wfmd]                       \[ [solution][wfmd_py] \]
| ALPH | [Alignment-Based Phylogeny][alph]                                      \[ [solution][alph_py] \]
| ASMQ | [Assessing Assembly Quality with N50 and N75][asmq]                    \[ [solution][asmq_py] \]
| CSET | [Fixing an Inconsistent Character Set][cset]                           \[ [solution][cset_py] \]
| EBIN | [Wright-Fisher's Expected Behavior][ebin]                              \[ [solution][ebin_py] \]
| FOUN | [The Founder Effect and Genetic Drift][foun]                           \[ [solution][foun_py] \]
| GAFF | [Global Alignment with Scoring Matrix and Affine Gap Penalty][gaff]    \[ [solution][gaff_py] \]
| GREP | [Genome Assembly with Perfect Coverage and Repeats][grep]              \[ [solution][grep_py] \]
| OAP  | [Overlap Alignment][oap]                                               \[ [solution][oap_py] \]
| QRTD | [Quartet Distance][qrtd]                                               \[ [solution][qrtd_py] \]
| SIMS | [Finding a Motif with Modifications][sims]                             \[ [solution][sims_py] \]
| SMGB | [Semiglobal Alignment][smgb]                                           \[ [solution][smgb_py] \]
| KSIM | [Finding All Similar Motifs][ksim]                                     \[ [solution][ksim_py] \]
| LAFF | [Local Alignment with Affine Gap Penalty][laff]                        \[ [solution][laff_py] \]
| OSYM | [Isolating Symbols in Alignments][osym]                                \[ [solution][osym_py] \]
| RSUB | [Identifying Reversing][rsub]                                          \[ [solution][rsub_py] \]


<!-- &#x1f512;   for the lock -->
<!-- tiltle -->
[dna]: http://rosalind.info/problems/dna/
[rna]: http://rosalind.info/problems/rna/
[revc]: http://rosalind.info/problems/revc/
[hamm]: http://rosalind.info/problems/hamm/
[prot]: http://rosalind.info/problems/prot/
[subs]: http://rosalind.info/problems/subs/
[perm]: http://rosalind.info/problems/perm/
[prtm]: http://rosalind.info/problems/prtm/
[fib]: http://rosalind.info/problems/fib/
[gc]: http://rosalind.info/problems/gc/
[iprb]: http://rosalind.info/problems/iprb/
[lcsm]: http://rosalind.info/problems/lcsm/
[mprt]: http://rosalind.info/problems/mprt/
[mrna]: http://rosalind.info/problems/mrna/
[orf]: http://rosalind.info/problems/orf/
[revp]: http://rosalind.info/problems/revp/
[splc]: http://rosalind.info/problems/splc/
[lexf]: http://rosalind.info/problems/lexf/
[sign]: http://rosalind.info/problems/sign/
[cons]: http://rosalind.info/problems/cons/
[fibd]: http://rosalind.info/problems/fibd/
[grph]: http://rosalind.info/problems/grph/
[iev]: http://rosalind.info/problems/iev/
[lia]: http://rosalind.info/problems/lia/
[lgis]: http://rosalind.info/problems/lgis/
[long]: http://rosalind.info/problems/long/
[pmch]: http://rosalind.info/problems/pmch/
[pper]: http://rosalind.info/problems/pper/
[prob]: http://rosalind.info/problems/prob/
[sseq]: http://rosalind.info/problems/sseq/
[tran]: http://rosalind.info/problems/tran/
[tree]: http://rosalind.info/problems/tree/
[cat]: http://rosalind.info/problems/cat/
[corr]: http://rosalind.info/problems/corr/
[inod]: http://rosalind.info/problems/inod/
[kmer]: http://rosalind.info/problems/kmer/
[kmp]: http://rosalind.info/problems/kmp/
[lcsq]: http://rosalind.info/problems/lcsq/
[lexv]: http://rosalind.info/problems/lexv/
[mmch]: http://rosalind.info/problems/mmch/
[pdst]: http://rosalind.info/problems/pdst/
[rear]: http://rosalind.info/problems/rear/
[rstr]: http://rosalind.info/problems/rstr/
[sset]: http://rosalind.info/problems/sset/
[aspc]: http://rosalind.info/problems/aspc/
[edit]: http://rosalind.info/problems/edit/
[eval]: http://rosalind.info/problems/eval/
[motz]: http://rosalind.info/problems/motz/
[nwck]: http://rosalind.info/problems/nwck/
[scsp]: http://rosalind.info/problems/scsp/
[seto]: http://rosalind.info/problems/seto/
[sort]: http://rosalind.info/problems/sort/
[spec]: http://rosalind.info/problems/spec/
[trie]: http://rosalind.info/problems/trie/
[conv]: http://rosalind.info/problems/conv/
[ctbl]: http://rosalind.info/problems/ctbl/
[dbru]: http://rosalind.info/problems/dbru/
[edta]: http://rosalind.info/problems/edta/
[full]: http://rosalind.info/problems/full/
[indc]: http://rosalind.info/problems/indc/
[itwv]: http://rosalind.info/problems/itwv/
[lrep]: http://rosalind.info/problems/lrep/
[nkew]: http://rosalind.info/problems/nkew/
[rnas]: http://rosalind.info/problems/rnas/
[afrq]: http://rosalind.info/problems/afrq/
[cstr]: http://rosalind.info/problems/cstr/
[ctea]: http://rosalind.info/problems/ctea/
[cunr]: http://rosalind.info/problems/cunr/
[glob]: http://rosalind.info/problems/glob/
[pcov]: http://rosalind.info/problems/pcov/
[prsm]: http://rosalind.info/problems/prsm/
[qrt]: http://rosalind.info/problems/qrt/
[sgra]: http://rosalind.info/problems/sgra/
[suff]: http://rosalind.info/problems/suff/
[chbp]: http://rosalind.info/problems/chbp/
[cntq]: http://rosalind.info/problems/cntq/
[eubt]: http://rosalind.info/problems/eubt/
[gasm]: http://rosalind.info/problems/gasm/
[gcon]: http://rosalind.info/problems/gcon/
[ling]: http://rosalind.info/problems/ling/
[loca]: http://rosalind.info/problems/loca/
[mend]: http://rosalind.info/problems/mend/
[mgap]: http://rosalind.info/problems/mgap/
[mrep]: http://rosalind.info/problems/mrep/
[mult]: http://rosalind.info/problems/mult/
[pdpl]: http://rosalind.info/problems/pdpl/
[root]: http://rosalind.info/problems/root/
[sexl]: http://rosalind.info/problems/sexl/
[sptd]: http://rosalind.info/problems/sptd/
[wfmd]: http://rosalind.info/problems/wfmd/
[alph]: http://rosalind.info/problems/alph/
[asmq]: http://rosalind.info/problems/asmq/
[cset]: http://rosalind.info/problems/cset/
[ebin]: http://rosalind.info/problems/ebin/
[foun]: http://rosalind.info/problems/foun/
[gaff]: http://rosalind.info/problems/gaff/
[grep]: http://rosalind.info/problems/grep/
[oap]: http://rosalind.info/problems/oap/
[qrtd]: http://rosalind.info/problems/qrtd/
[sims]: http://rosalind.info/problems/sims/
[smgb]: http://rosalind.info/problems/smgb/
[ksim]: http://rosalind.info/problems/ksim/
[laff]: http://rosalind.info/problems/laff/
[osym]: http://rosalind.info/problems/osym/
[rsub]: http://rosalind.info/problems/rsub/

<!-- solution -->
[dna_py]: https://github.com/thongle91/rosalind/tree/master/codes/dna.py
[rna_py]: https://github.com/thongle91/rosalind/tree/master/codes/rna.py
[revc_py]: https://github.com/thongle91/rosalind/tree/master/codes/revc.py
[hamm_py]: https://github.com/thongle91/rosalind/tree/master/codes/hamm.py
[prot_py]: https://github.com/thongle91/rosalind/tree/master/codes/prot.py
[subs_py]: https://github.com/thongle91/rosalind/tree/master/codes/subs.py
[perm_py]: https://github.com/thongle91/rosalind/tree/master/codes/perm.py
[prtm_py]: https://github.com/thongle91/rosalind/tree/master/codes/prtm.py
[fib_py]: https://github.com/thongle91/rosalind/tree/master/codes/fib.py
[gc_py]: https://github.com/thongle91/rosalind/tree/master/codes/gc.py
[iprb_py]: https://github.com/thongle91/rosalind/tree/master/codes/iprb.py
[lcsm_py]: https://github.com/thongle91/rosalind/tree/master/codes/lcsm.py
[mprt_py]: https://github.com/thongle91/rosalind/tree/master/codes/mprt.py
[mrna_py]: https://github.com/thongle91/rosalind/tree/master/codes/mrna.py
[orf_py]: https://github.com/thongle91/rosalind/tree/master/codes/orf.py
[revp_py]: https://github.com/thongle91/rosalind/tree/master/codes/revp.py
[splc_py]: https://github.com/thongle91/rosalind/tree/master/codes/splc.py
[lexf_py]: https://github.com/thongle91/rosalind/tree/master/codes/lexf.py
[sign_py]: https://github.com/thongle91/rosalind/tree/master/codes/sign.py
[cons_py]: https://github.com/thongle91/rosalind/tree/master/codes/cons.py
[fibd_py]: https://github.com/thongle91/rosalind/tree/master/codes/fibd.py
[grph_py]: https://github.com/thongle91/rosalind/tree/master/codes/grph.py
[iev_py]: https://github.com/thongle91/rosalind/tree/master/codes/iev.py
[lia_py]: https://github.com/thongle91/rosalind/tree/master/codes/lia.py
[lgis_py]: https://github.com/thongle91/rosalind/tree/master/codes/lgis.py
[long_py]: https://github.com/thongle91/rosalind/tree/master/codes/long.py
[pmch_py]: https://github.com/thongle91/rosalind/tree/master/codes/pmch.py
[pper_py]: https://github.com/thongle91/rosalind/tree/master/codes/pper.py
[prob_py]: https://github.com/thongle91/rosalind/tree/master/codes/prob.py
[sseq_py]: https://github.com/thongle91/rosalind/tree/master/codes/sseq.py
[tran_py]: https://github.com/thongle91/rosalind/tree/master/codes/tran.py
[tree_py]: https://github.com/thongle91/rosalind/tree/master/codes/tree.py
[cat_py]: https://github.com/thongle91/rosalind/tree/master/codes/cat.py
[corr_py]: https://github.com/thongle91/rosalind/tree/master/codes/corr.py
[inod_py]: https://github.com/thongle91/rosalind/tree/master/codes/inod.py
[kmer_py]: https://github.com/thongle91/rosalind/tree/master/codes/kmer.py
[kmp_py]: https://github.com/thongle91/rosalind/tree/master/codes/kmp.py
[lcsq_py]: https://github.com/thongle91/rosalind/tree/master/codes/lcsq.py
[lexv_py]: https://github.com/thongle91/rosalind/tree/master/codes/lexv.py
[mmch_py]: https://github.com/thongle91/rosalind/tree/master/codes/mmch.py
[pdst_py]: https://github.com/thongle91/rosalind/tree/master/codes/pdst.py
[rear_py]: https://github.com/thongle91/rosalind/tree/master/codes/rear.py
[rstr_py]: https://github.com/thongle91/rosalind/tree/master/codes/rstr.py
[sset_py]: https://github.com/thongle91/rosalind/tree/master/codes/sset.py
[aspc_py]: https://github.com/thongle91/rosalind/tree/master/codes/aspc.py
[edit_py]: https://github.com/thongle91/rosalind/tree/master/codes/edit.py
[eval_py]: https://github.com/thongle91/rosalind/tree/master/codes/eval.py
[motz_py]: https://github.com/thongle91/rosalind/tree/master/codes/motz.py
[nwck_py]: https://github.com/thongle91/rosalind/tree/master/codes/nwck.py
[scsp_py]: https://github.com/thongle91/rosalind/tree/master/codes/scsp.py
[seto_py]: https://github.com/thongle91/rosalind/tree/master/codes/seto.py
[sort_py]: https://github.com/thongle91/rosalind/tree/master/codes/sort.py
[spec_py]: https://github.com/thongle91/rosalind/tree/master/codes/spec.py
[trie_py]: https://github.com/thongle91/rosalind/tree/master/codes/trie.py
[conv_py]: https://github.com/thongle91/rosalind/tree/master/codes/conv.py
[ctbl_py]: https://github.com/thongle91/rosalind/tree/master/codes/ctbl.py
[dbru_py]: https://github.com/thongle91/rosalind/tree/master/codes/dbru.py
[edta_py]: https://github.com/thongle91/rosalind/tree/master/codes/edta.py
[full_py]: https://github.com/thongle91/rosalind/tree/master/codes/full.py
[indc_py]: https://github.com/thongle91/rosalind/tree/master/codes/indc.py
[itwv_py]: https://github.com/thongle91/rosalind/tree/master/codes/itwv.py
[lrep_py]: https://github.com/thongle91/rosalind/tree/master/codes/lrep.py
[nkew_py]: https://github.com/thongle91/rosalind/tree/master/codes/nkew.py
[rnas_py]: https://github.com/thongle91/rosalind/tree/master/codes/rnas.py
[afrq_py]: https://github.com/thongle91/rosalind/tree/master/codes/afrq.py
[cstr_py]: https://github.com/thongle91/rosalind/tree/master/codes/cstr.py
[ctea_py]: https://github.com/thongle91/rosalind/tree/master/codes/ctea.py
[cunr_py]: https://github.com/thongle91/rosalind/tree/master/codes/cunr.py
[glob_py]: https://github.com/thongle91/rosalind/tree/master/codes/glob.py
[pcov_py]: https://github.com/thongle91/rosalind/tree/master/codes/pcov.py
[prsm_py]: https://github.com/thongle91/rosalind/tree/master/codes/prsm.py
[qrt_py]: https://github.com/thongle91/rosalind/tree/master/codes/qrt.py
[sgra_py]: https://github.com/thongle91/rosalind/tree/master/codes/sgra.py
[suff_py]: https://github.com/thongle91/rosalind/tree/master/codes/suff.py
[chbp_py]: https://github.com/thongle91/rosalind/tree/master/codes/chbp.py
[cntq_py]: https://github.com/thongle91/rosalind/tree/master/codes/cntq.py
[eubt_py]: https://github.com/thongle91/rosalind/tree/master/codes/eubt.py
[gasm_py]: https://github.com/thongle91/rosalind/tree/master/codes/gasm.py
[gcon_py]: https://github.com/thongle91/rosalind/tree/master/codes/gcon.py
[ling_py]: https://github.com/thongle91/rosalind/tree/master/codes/ling.py
[loca_py]: https://github.com/thongle91/rosalind/tree/master/codes/loca.py
[mend_py]: https://github.com/thongle91/rosalind/tree/master/codes/mend.py
[mgap_py]: https://github.com/thongle91/rosalind/tree/master/codes/mgap.py
[mrep_py]: https://github.com/thongle91/rosalind/tree/master/codes/mrep.py
[mult_py]: https://github.com/thongle91/rosalind/tree/master/codes/mult.py
[pdpl_py]: https://github.com/thongle91/rosalind/tree/master/codes/pdpl.py
[root_py]: https://github.com/thongle91/rosalind/tree/master/codes/root.py
[sexl_py]: https://github.com/thongle91/rosalind/tree/master/codes/sexl.py
[sptd_py]: https://github.com/thongle91/rosalind/tree/master/codes/sptd.py
[wfmd_py]: https://github.com/thongle91/rosalind/tree/master/codes/wfmd.py
[alph_py]: https://github.com/thongle91/rosalind/tree/master/codes/alph.py
[asmq_py]: https://github.com/thongle91/rosalind/tree/master/codes/asmq.py
[cset_py]: https://github.com/thongle91/rosalind/tree/master/codes/cset.py
[ebin_py]: https://github.com/thongle91/rosalind/tree/master/codes/ebin.py
[foun_py]: https://github.com/thongle91/rosalind/tree/master/codes/foun.py
[gaff_py]: https://github.com/thongle91/rosalind/tree/master/codes/gaff.py
[grep_py]: https://github.com/thongle91/rosalind/tree/master/codes/grep.py
[oap_py]: https://github.com/thongle91/rosalind/tree/master/codes/oap.py
[qrtd_py]: https://github.com/thongle91/rosalind/tree/master/codes/qrtd.py
[sims_py]: https://github.com/thongle91/rosalind/tree/master/codes/sims.py
[smgb_py]: https://github.com/thongle91/rosalind/tree/master/codes/smgb.py
[ksim_py]: https://github.com/thongle91/rosalind/tree/master/codes/ksim.py
[laff_py]: https://github.com/thongle91/rosalind/tree/master/codes/laff.py
[osym_py]: https://github.com/thongle91/rosalind/tree/master/codes/osym.py
[rsub_py]: https://github.com/thongle91/rosalind/tree/master/codes/rsub.py